presentation/frontend/src/presentation-frontend/PresentationManager.ts

```diff
@@ -77,13 +77,13 @@ export class PresentationManager implements IDisposable {
    * An event raised when hierarchies created using specific ruleset change
    * @alpha
    */
-  public onIModelHierarchyChanged = new BeEvent<(args: { ruleset: Ruleset, updateInfo: HierarchyUpdateInfo }) => void>();
+  public onIModelHierarchyChanged = new BeEvent<(args: { rulesetId: string, updateInfo: HierarchyUpdateInfo, imodelKey: string }) => void>();

```

Comment:
I'm thinking maybe it's worth extracting args type into a separate interface - that would allow us to add docs for the attributes. E.g. I'd like `imodelKey` docs to mention that it matches `[[IModelConnection.key]]`

Same for `onIModelContentChanged`

Reply:
Created types for change events args